Automattic/Simplecons
# Optimise PDFs

This is a PR to optimise Simplecons PDFs. The goal of optimisation is to reduce the file sizes of all the PDFs in the Simplecons library.

### To Test
- Verify that the optimised icons have not changed visually compared with their non-optimsed versions.
- Verify that the icons work as intended in the app.
